toml.md

```diff
@@ -759,8 +759,9 @@ name = "Regina Dogman"
 member_since = 1999-08-04
 ```
 
-Dotted keys create and define a table for each key part before the last one,
-provided that such tables were not previously created.
+Dotted keys create and define a table for each key part before the last one. Any
+such table must have all its key/value pairs defined under the current `[table]`
+header, or in the root table if defined before all headers.
```

Comment:
```suggestion
header, or in the root table if defined before all headers, or in one inline table.
```

@ChristianSi thank you! I think this change would have helped me. At that point I wonder if your concept of block should be defined in the spec, it seems a clearer way to express that behavior.